rules-tests/TypeDeclaration/Rector/ClassMethod/ReturnTypeFromStrictNativeCallRector/Fixture/native_method_call_returning_union_false.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNativeCallRector\Fixture;
+
+final class SomeNativeMethodCall
+{
+    public function run()
+    {
+        return ob_get_clean();
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\ReturnTypeFromStrictNativeCallRector\Fixture;
+
+final class SomeNativeMethodCall
+{
+    public function run(): string|false
+    {
+        return ob_get_clean();
```

Comment:
could you add 1 more fixture?

```
if (rand(0,1)) {
    return ob_get_clean();
}

return in_array($a, $b, true)

Reply:
I've added 2 fixtures because the result depends on the order in which the functions are called.